server/http_handler.go

```diff
@@ -518,6 +519,16 @@ func (t *tikvHandlerTool) getRegionsMeta(regionIDs []uint64) ([]RegionMeta, erro
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
+
+		failpoint.Inject("errGetRegionByIDEmpty", func(val failpoint.Value) {
+			if val.(bool) {
+				meta = nil
```

Comment:
Why not `return` directly?

Reply:
I wanna inject the case that meta is nil.

Reply:
Is it better to inject it in `GetRegionByID`?

Reply:
@jackysp  `GetRegionByID` function is in `PD`, How to inject?

Reply:
I think there is another implement in `mocktikv`.